pkg/restapi/oidc/operations.go

```diff
@@ -386,3 +411,70 @@ func (o *Operation) userLogoutHandler(w http.ResponseWriter, r *http.Request) {
 
 	logger.Debugf("finished handling logout request")
 }
+
+func (o *Operation) onboardUser(sub string) error {
+	b := make([]byte, 32)
+
+	_, err := rand.Read(b)
+	if err != nil {
+		return fmt.Errorf("create user secret key : %w", err)
+	}
+
+	secrets, err := o.secretSplitter.Split(b, 2, 2)
+	if err != nil {
+		return fmt.Errorf("split user secret key : %w", err)
+	}
+
+	// TODO https://github.com/trustbloc/edge-agent/issues/488 send half secret key to hub-auth and remove logger
+	logger.Errorf(string(secrets[0]))
+	logger.Errorf(string(secrets[0]))
+
+	reqBytes, err := json.Marshal(createKeystoreReq{
+		Controller: sub,
+	})
+	if err != nil {
+		return fmt.Errorf("marshal create keystore req : %w", err)
+	}
+
+	req, err := http.NewRequestWithContext(context.TODO(),
```

Comment:
access_token added in followup?

Reply:
Yes, updated the TODO [comment](https://github.com/trustbloc/edge-agent/pull/490/files#diff-96466bce0ffd4cb8bd674beee84455ded8463ad6bdf53e49635ef6c333eddc66R445).